docs/book/stacks-and-components/stack-deployment/cloud-stacks/minimal-gcp-stack.md

```diff
@@ -0,0 +1,179 @@
+---
+description: A simple guide to quickly set up a minimal stack on GCP.
+---
+
+# Set up minimal GCP stack
+
+Here's a 7-step guide to get a production-ready GCP stack.
+
+
+
+### 1) Choose a GCP project&#x20;
+
+In the Google Cloud console, on the project selector page, select or [create a Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+{% hint style="info" %}
+If you don't plan to keep the resources that you create in this procedure, create a new project. After you finish these steps, you can delete the project, thereby removing all resources associated with the project.
+{% endhint %}
+
+### 2) Enable GCloud APIs
+
+The [following APIs](https://console.cloud.google.com/flows/enableapi?apiid=cloudfunctions,cloudbuild.googleapis.com,artifactregistry.googleapis.com,run.googleapis.com,logging.googleapis.com\\\&redirect=https://cloud.google.com/functions/docs/create-deploy-gcloud&\\\_ga=2.103703808.1862683951.1694002459-205697788.1651483076&\\\_gac=1.161946062.1694011263.Cj0KCQjwxuCnBhDLARIsAB-cq1ouJZlVKAVPMsXnYrgQVF2t1Q2hUjgiHVpHXi2N0NlJvG3j3y-PPh8aAoSIEALw\\\_wcB) will need to be enabled within your chosen gcp project.
+
+* Cloud Functions API
+* Cloud Build API
+* Artifact Registry API
+* Cloud Run Admin API
+* Cloud Logging API
+
```

Comment:
Maybe just explain here why people did that + why those APIs are needed? (briefly)